exec/csnex/csnex.cs

```diff
@@ -76,12 +76,16 @@ static int Main(string[] args)
             fs.Read(code, 0, (int)nSize);
             fs.Close();
 
-            Bytecode bc = new Bytecode();
-            bc.LoadBytecode(gOptions.Filename, code);
-
-            Executor exec = new Executor(bc);
-            retval = exec.Run(gOptions.EnableAssertions);
+            try {
+                Bytecode bc = new Bytecode();
+                bc.LoadBytecode(gOptions.Filename, code);
 
+                Executor exec = new Executor(bc);
+                retval = exec.Run(gOptions.EnableAssertions);
+            } catch (Exception ex) {
+                Console.Error.WriteLine(ex.Message);
+                return -1;
```

Comment:
-1 is not a valid process exit code. Use 1.

Reply:
Fixed.